dashboard/Dashboard.py

```diff
@@ -10,7 +8,9 @@
 import streamlit as st
 import nltk
 
-from functions import create_donut_chart, create_genre_bar_chart, create_line_chart, create_os_bar_chart, create_platform_bar_chart, create_release_line_chart, create_word_cloud, preprocess_descriptions
+from functions import (create_donut_chart, create_genre_bar_chart, create_line_chart,
```

Comment:
Would it be better just to import the whole file since your importing a lot of its functionality.

Reply:
Pylint doesn't like wildcard imports, but this warning could always be disabled.